packages/orbit-components/package.json

```diff
@@ -73,8 +73,8 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.21.0",
-    "react": ">=18.3.0",
-    "react-dom": ">=18.3.0"
+    "react": ">=19.1.0",
+    "react-dom": ">=19.1.0"
```

Comment:
Does it need to be 19.1? Not a strong opinion, just curious because in frontend/core we set ^19.0.0

Reply:
I don't think so, no. Will change